### PR TITLE
Yahoo amp-ad plugin

### PR DESCRIPTION
**Pull request by src-code**
Adds amp-ad support for Yahoo display ads.  

Note that ads are rendered in a nested iframe, so I'm not making use of `renderStart()` or `noContentAvailable()` here.  

This replaces PR #5122.

cc @lannka

### CODE REVIEW

**Comment by src-code** [comment]
Doesn't pass `gulp check-types` - I'll get rid of the global function call.

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Webediads](../../ads/webediads.md)
 - [Weborama](../../ads/weborama.md)
 - [Widespace](../../ads/widespace.md)
+- [Yahoo](../ads/yahoo.md)
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: one more `../`

File: ads/_config.js [hunk 2]
```diff
@@ -382,6 +382,11 @@ export const adConfig = {
 
   widespace: {},
 
+  yahoo: {
+    prefetch: 'https://s.yimg.com/os/ampad/display.js',
+    preconnect: 'https://us.adserver.yahoo.com',
```
Comment (lannka) [inline ads/_config.js]: i don't see this host used in this change. i assume it's used in loaded JS?

File: extensions/amp-ad/amp-ad.md (hunk 1 above)
Comment (src-code) [inline extensions/amp-ad/amp-ad.md]: Oops!

File: ads/_config.js (hunk 2 above)
Comment (src-code) [inline ads/_config.js]: Yes, that's correct

**Pull request merged**